stdlib/universe/reduce.go

```diff
@@ -134,7 +136,7 @@ func createReduceTransformation(id execute.DatasetID, mode execute.AccumulationM
 	}
 	cache := execute.NewTableBuilderCache(a.Allocator())
 	d := execute.NewDataset(id, mode, cache)
-	t, err := NewReduceTransformation(d, cache, s)
+	t, err := NewReduceTransformation(d, cache, s, a.Context(), a.Dependencies()["interpreter"].(dependencies.Interface))
```

Comment:
we should be able to change this to executetest.NewTestExecuteDependencies()

Reply:
same here, not test code.